falcon/status_codes.py

```diff
@@ -115,6 +127,14 @@
 HTTP_GATEWAY_TIMEOUT = HTTP_504
 HTTP_505 = '505 HTTP Version not supported'
 HTTP_HTTP_VERSION_NOT_SUPPORTED = HTTP_505
+HTTP_506 = '506 Variant Also Negotiates'
```

Comment:
506 is from a very old experimental RFC. Are there any examples that demonstrate modern usage of this status? Otherwise I'm inclined to say it doesn't need to be included in the framework.